prog/pwm/fancontrol

```diff
@@ -390,6 +393,39 @@ function pwmdisable()
 		return 0
 	fi
 
+	# Try to restore pwmN and pwmN_enable value to the same state as before fancontrol start
+	# restoring the pwmN value is tried first, before the pwmN_enable mode switch. Some chips seem to need this to properly restore fan operation, when activating automatic (2) mode.
+	if [ -v PWMORIGVALUES[${1}] ]
+	then
+		local PWM_ENABLE_ORIG_STATE=`echo ${PWMORIGVALUES[${1}]} | cut -d ';' -f1`
+		local PWM_ORIG_STATE=`echo ${PWMORIGVALUES[${1}]} | cut -d ';' -f2`
+		#restore the original pwmN value
+		if [ "$DEBUG" != "" ]
+		then
+			echo "Restoring ${1} original value of $PWM_ORIG_STATE"
+		fi
+		echo $PWM_ORIG_STATE > ${1} 2> /dev/null
+        #restore the pwmN_enable value, if it is not 1. 1 is already set through fancontrol and setting it again might just reset the pwmN value.
```

Comment:
I can't see the entire comment here in the review. Please split into multiple lines.